src/GObject.php

```diff
@@ -223,7 +223,7 @@ private static function getMarshaler(string $name, callable $callback): ?Closure
                         CData  $hint,
                         ?CData $data
                     ) use (&$callback): void {
-                        assert($numberOfParams === 0);
+                        assert($numberOfParams === 1);
```

Comment:
This was causing a test to fail.
I saw that the other asserts checked the param count based on the commented signature - 1, so it looks like this one should have been 1.

Reply:
> As to why it doesn't break in git actions, my guess is that they are running against an older version of libvips? (I noted the if (FFI::atLeast(8, 9)) { on line 217)

Actually, that doesn't seem to be the case. A slightly older successful build used libvips 8.12: https://github.com/libvips/php-vips/actions/runs/7757736861/job/21160084456#step:4:121

Maybe PHP is running in production mode in the GitHub actions, which disables assertions? If that's the case, @jcupitt shouldn't we enable enable them?